docs/support/compatibility.md

```diff
@@ -6,15 +6,15 @@ position: 190
 
 The table below outlines the backward compatibility between Octopus Server and related components
 
-| Octopus Server    | Octopus.Client Octo.exe | Calamari         | Tentacle      | TeamCity Plugin  |
-| --------------    | ----------------------- | ------------     | ------------  | ---------------  |
-| 3.2               | 3.2                     | 3.0 ➜ latest    | 3.2           |                  |
-| 3.3               | 3.3 ➜ 4.30.3           | 3.3              | 3.0 ➜ latest | 3.3 ➜ latest    |
-| 3.4               | 3.3 ➜ 4.30.3           | 3.4              | 3.0 ➜ latest | 3.3 ➜ latest    |
-| 3.5 ➜ 2018.2     | 3.3 ➜ 4.30.3           | 3.5 ➜ latest    | 3.0 ➜ latest | 3.3 ➜ latest    |
-| 2018.2 ➜ 2018.12 | 4.30.7 ➜ 4.47.0        | 3.5 ➜ latest    | 3.0 ➜ latest | 3.3 ➜ latest    |
-| 2019.1*           | 5.0.0 ➜ 5.2.7          | 3.5 ➜ latest    | 4.0 ➜ latest | 5.0 ➜ latest    |
-| 2019.2* ➜ latest | 6.0.0 ➜ latest         | 3.5 ➜ latest    | 4.0 ➜ latest | 5.0 ➜ latest    |
+| Octopus Server    | Octopus.Client & Octopus CLI (octo) | Calamari         | Tentacle      | TeamCity Plugin  |
```

Comment:
Now that Octopus.Client & Octopus CLI have different lifecycles... We should probably consider splitting these up.